server/src/main/java/com/epam/aidial/core/server/validation/ValidationUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.aidial.core.server.validation;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.ConstraintViolationException;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
+
+import java.util.Set;
+
+public class ValidationUtil {
```

Comment:
Please try to avoid unnecessary utility classes. The class is used in a single place. Move it to BeanDeserializerWithValidation

Reply:
moved